components/gamification/streak/ChatStreakDisplay.tsx

```diff
@@ -127,7 +127,8 @@ const styles = StyleSheet.create({
     flexDirection: 'row',
     justifyContent: 'center',
     alignItems: 'center',
-    flexWrap: 'wrap',
+    flexWrap: 'nowrap',
```

Comment:
This doesn't feel right. This may be working on your screen but it will likely fail on other screens. Let's keep it as wrap.

Reply:
I researched this and flexWrap: 'nowrap' was the suggested solution

Reply:
> The flex items are laid out in a single line which may cause the flex container to overflow. The cross-start is either equivalent to start or before depending on the [flex-direction](https://developer.mozilla.org/en-US/docs/Web/CSS/flex-direction) value. This is the default value.

I know you want to keep them on a single line, but this will overflow the values. We should keep it as `wrap`